### ISSUE TITLE
Transparent Background

### ISSUE THREAD

**Issue by zaun** [lecram/gifdec#5]
The attached image 1 has a transparent background, though when loaded through the lib, its getting a blueish background. The attached image 2 has a transparent background and it getting a black color. Image 3 has a transparent background and it getting white.

I'm thinking they should all be black.

image 1:
![test2](https://user-images.githubusercontent.com/1311097/62818138-8944bc00-bade-11e9-944d-9c1d6dcb724f.gif)

image 2:
![test1](https://user-images.githubusercontent.com/1311097/62818149-cc069400-bade-11e9-9ca7-74343c41b8cc.gif)

image 3:
![test3](https://user-images.githubusercontent.com/1311097/62818164-1720a700-badf-11e9-988d-2b112b937d3b.gif)

**Comment by zaun** [comment]
Is there a way for me to get the transparent color?

Basically when I take the `frame` from `gd_render_frame(gif, frame);` and try to display it, I don't know what pixels in the frame to skip drawing.

**Comment by lecram** [comment]
Hi @zaun. Sorry for the long delay.

After some frustrating attempts to solve this issue, today I learned that what GIF applications consider "transparent color" is simply what the GIF specification refers to as "background color".

A new function `gd_is_bgcolor()` has been added to help applications handle transparent pixels. It's explained in the new section 7 of the README, which hopefully answers your question of how to get the transparent color.

The file `example.c` has also been updated to handle GIFs with transparent background (shown as a checkerboard pattern).
